helpdesk_mgmt/README.rst

```diff
@@ -115,6 +115,8 @@ Tags
 Usage
 =====
 
+#. Different user permissions are available to restrict access to tickets: User: Personal tickets, User: Team tickets and User.
```

Comment:
This is configuration, not usage, and please explain the goal of each permission.

Reply:
IMO it is not necessary to add this because I think the name of the groups is obvious, what do you think?